cmd/opera/launcher/validator.go

```diff
@@ -33,38 +33,28 @@ var validatorPasswordFlag = cli.StringFlag{
 // command line flags or from the keystore if CLI indexed.
 func setValidator(ctx *cli.Context, cfg *emitter.Config) error {
 	// Extract the current validator address, new flag overriding legacy one
-	var validatorID idx.ValidatorID
-	var validatorPubkey validatorpk.PubKey
 	var err error
```

Comment:
I think we can remove err variable on L36 and use `err` variable inside if blocks.

Reply:
We would need 2 lines instead of 1 in this case (there's 2 if blocks which use `err`). Do you think we should do it?

Reply:
We can do like that:
```
	// var err error
	if ctx.GlobalIsSet(FakeNetFlag.Name) {
		var num idx.Validator
		id, num, err := parseFakeGen(ctx.GlobalString(FakeNetFlag.Name))
		if err != nil {
			return err
		}
		cfg.Validator.ID = id 
		validators := makegenesis.GetFakeValidators(num)
		cfg.Validator.PubKey = validators.Map()[cfg.Validator.ID].PubKey
	} 

	if ctx.GlobalIsSet(validatorIDFlag.Name) {
		cfg.Validator.ID = idx.ValidatorID(ctx.GlobalInt(validatorIDFlag.Name))
	}
	if ctx.GlobalIsSet(validatorPubkeyFlag.Name) {
		pk, err := validatorpk.FromString(ctx.GlobalString(validatorPubkeyFlag.Name))
		if err != nil {
			return err
		}
		cfg.Validator.PubKey = pk
	}
```

Reply:
Yeah, looking good